samples/wifi/radio_test/radio_test_subcommands.rst

```diff
@@ -19,142 +19,221 @@ Wi-Fi radio test subcommands
 
    * - Subcommand
      - Argument
+     - Default value
+     - Type
      - Description
+   * - set_defaults
+     - N/A
+     - N/A
+     - Configuration
+     - Reset all configuration parameters to their default values.
    * - phy_calib_rxdc
      - | 0 - Disable
        | 1 - Enable
+     - 1
+     - Configuration
      - Enable/Disable RX DC calibration.
    * - phy_calib_txdc
      - | 0 - Disable
        | 1 - Enable
+     - 1
+     - Configuration
      - Enable/Disable TX DC calibration.
    * - phy_calib_txpow
      - | 0 - Disable
        | 1 - Enable
+     - 0
+     - Configuration
      - Enable/Disable TX power calibration.
    * - phy_calib_rxiq
      - | 0 - Disable
        | 1 - Enable
+     - 1
+     - Configuration
      - Enable/Disable RX IQ calibration.
    * - phy_calib_txiq
      - | 0 - Disable
        | 1 - Enable
+     - 1
+     - Configuration
      - Enable/Disable TX IQ calibration.
    * - he_ltf
      - | 0 - 1x HE LTF
        | 1 - 2x HE LTF
        | 2 - 4x HE LTF
+     - 2
+     - Configuration
      - Configure HE long training field (LTF) value while transmitting the packet.
    * - he_gi
      - | 0 - 0.8 us
        | 1 - 1.6 us
        | 2 - 3.2 us
+     - 2
+     - Configuration
      - Configure HE guard interval (GI) while transmitting the packet.
-   * - rf_params
-     - Hex value string
-     - Hexadecimal value string for RF related parameters. See :ref:`wifi_radio_test_rf_params` for the description.
    * - tx_pkt_tput_mode
      - | 0 - Legacy
        | 1 - HT mode
        | 2 - VHT mode
        | 3 - HE (SU) mode
        | 4 - HE (ERSU) mode
        | 5 - HE (TB) mode
+     - 0
+     - Configuration
      - Throughput mode to be used for transmitting the packet.
    * - tx_pkt_sgi
      - | 0 - Disable
        | 1 - Enable
+     - 0
+     - Configuration
      - Enable/Disable Short guard interval (GI) while transmitting the packet.
    * - tx_pkt_preamble
      - | 0 - Short Preamble
        | 1 - Long Preamble
        | 2 - Mixed Preamble
+     - 1
+     - Configuration
      - Type of preamble to be used for each packet. Short/Long Preamble are applicable only when tx_pkt_tput_mode is set to Legacy and Mixed Preamble is applicable only when tx_pkt_tput_mode is set to HT/VHT.
    * - tx_pkt_mcs
      - | -1 - Not being used
        | <val> - MCS index to be used
+     - -1
+     - Configuration
      - MCS index at which TX packet will be transmitted. Mutually exclusive with tx_pkt_rate.
    * - tx_pkt_rate
      - | -1 - Not being used
        | <val> - Legacy rate to be used (1, 2, 5.5, 11, 6, 9, 12, 18, 24, 36, 48, 54)
+     - 6
+     - Configuration
      - Legacy rate at which packets will be transmitted. Mutually exclusive with tx_pkt_mcs.
    * - tx_pkt_gap
-     - <val> - (Min: 200, Max: 200000, Default: 200)
+     - <val> - (Min: 200, Max: 200000)
+     - 200
+     - Configuration
      - Interval between TX packets in microseconds.
-   * - chnl_primary
```

Comment:
description of API "init" has not been added

Reply:
Invalid